historical/security_group/differ.py

```diff
@@ -10,25 +10,20 @@
 
 from raven_python_lambda import RavenLambdaWrapper
 
+from historical.models import EPHEMERAL_PATHS
 from historical.common.dynamodb import process_dynamodb_record
 from historical.security_group.models import DurableSecurityGroupModel
 
 logging.basicConfig()
 log = logging.getLogger('historical')
 log.setLevel(logging.WARNING)
 
-EPHEMERAL_PATHS = [
-    "root['attribute_values']['eventTime']",
-    "root['attribute_values']['principalId']",
-    "root['attribute_values']['userIdentity']",
-]
-
 
 def is_new_revision(latest_revision, current_revision):
     """Determine if two revisions have actually changed."""
     diff = DeepDiff(
-        current_revision.__dict__,
-        latest_revision.__dict__,
+        current_revision._get_json(),
```

Comment:
@mikegrima I've modified this logic as I found the DeepDiff doesn't actually look inside of attributes to determine differences. Top level values work ok. This function returns plain dicts that we can then accurately diff. Let me know if you see any issues with this.